scripts/mkcloud

```diff
@@ -774,9 +776,59 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+    $scp root@$adminip:tempest.subunit.log "$artifacts_dir/tempest.subunit.log"
+
+    # Register the cloud in Rally and import the results
+    if [[ $rally_server ]] ; then
```

Comment:
maybe only execute this block if also the required artefacts are there ? qa_crowbarsetup.sh seems to respect want_tempest=0 as an environment variable which means you might not always have subunit log..